### PR TITLE
#26 command-line param to process only a specific collection(s)

### PR DESCRIPTION
**Pull request by maciejzgadzaj**
Ok, so out of sheer curiosity I had a look at how complicated it would be to achieve what I was thinking about originally, and apparently it was not that complicated at all. 

Bear with me though please, this is the very first time I am doing anything in Python, so there might be some silly things in the code. But then again, from quick tests it seems doing what I wanted it to do, meaning:

- `python3 plex_meta_manager.py` will process all defined collections
- `python3 plex_meta_manager.py --collections='Monty Python'` will process only `Monty Python` collection
- `python3 plex_meta_manager.py --collections=Action,Adventure` will process only `Action` and `Adventure` collections
- `python3 plex_meta_manager.py --collections=missing` (where there is no `missing` collection) will complain about `No collection to update`

Not really asking you here to merge it (although be my guest it you want), just wanted to share, and perhaps even ask for a code review if that's ok?

### CODE REVIEW

File: modules/config.py [hunk 1]
```diff
@@ -420,12 +420,13 @@ def update_libraries(self, test):
             logger.info("")
             util.seperator("{} Library {}Collections".format(library.name, "Test " if test else ""))
             collections = library.collections
-            if collections:
+            collections_to_process = (collections.keys() & requested_collections.split(",")) if requested_collections else collections
```
Comment (meisnate12) [inline modules/config.py]: You can use my util.get_list method to make sure the command line arg will work with "," or ", ".
```python
collections_to_process = (collections.keys() & util.get_list(requested_collections)) if requested_collections else collections
```

**Review by meisnate12 — CHANGES_REQUESTED** [review]
This looks good if resolve the conflicts I'll add this into the release i don't mind giving people more ways to run it.

**Comment by maciejzgadzaj** [comment]
Rebased, conflict resolved, `util.get_list()` used.

**Comment by doctarrwho** [comment]
Thanks @maciejzgadzaj and @meisnate12... Tested in develop and it works as advertised. This is very useful for one-off reprocessing of a collection.

**Comment by doctarrwho** [comment]
Spoke too soon! It was working last night, but I saw it fail this morning for the same collection.

```
[2021-02-17 10:34:30,428] [util.py:586]               [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "/mnt/c/repos/unraid/Plex-Meta-Manager/modules/config.py", line 457, in update_libraries
                                                                 |     collectionless = "plex_collectionless" in collections[c]
                                                                 | TypeError: 'set' object is not subscriptable
                                                                 |
[2021-02-17 10:34:30,429] [config.py:1119]            [ERROR]    | Unknown Error: 'set' object is not subscriptable
````

**Comment by meisnate12** [comment]
ok update and try again

**Comment by doctarrwho** [comment]
Fast as always! It works now 👍

**Pull request merged**